hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/client/ECReplicationConfig.java

```diff
@@ -89,7 +89,8 @@ public ECReplicationConfig(int data, int parity, EcCodec codec,
    *     XOR-10-4-4096K
    * IllegalArgumentException will be thrown if the passed string does not
    * match the defined pattern.
-   * @param string
+   * @param string Parameters used to create ECReplicationConfig instances.
+   *         Format: <codec>-<data>-<parity>-<chunksize>
```

Comment:
Javadoc allows HTML tags, so `<` and `>` should be written as `&lt;` and `&gt;`, otherwise this will be rendered as `----`.

Reply:
Thanks @adoroszlai , I'll update soon.